include/fluent-bit/flb_output.h

```diff
@@ -952,6 +960,102 @@ struct flb_output_flush *flb_output_flush_create(struct flb_task *task,
                 return NULL;
             }
 
+            out_flush->processed_event_chunk = flb_event_chunk_create(
+                                                evc->type,
+                                                0,
+                                                evc->tag,
+                                                flb_sds_len(evc->tag),
+                                                p_buf,
+                                                p_size);
+
+            if (out_flush->processed_event_chunk == NULL) {
+                flb_coro_destroy(coro);
+                flb_free(out_flush);
+                flb_free(p_buf);
+
+                return NULL;
+            }
+        }
+        else if (evc->type == FLB_EVENT_TYPE_PROFILES) {
+            p_buf = flb_calloc(evc->size * 2, sizeof(char));
+
+            if (p_buf == NULL) {
+                flb_coro_destroy(coro);
+                flb_free(out_flush);
+
+                return NULL;
+            }
+
+            p_size = evc->size;
+
+            chunk_offset = 0;
+            serialization_buffer_offset = 0;
+
+            while ((ret = cprof_decode_msgpack_create(
+                            &profile_context,
+                            (unsigned char *) evc->data,
+                            evc->size,
+                            &chunk_offset)) == CPROF_DECODE_MSGPACK_SUCCESS) {
+                ret = flb_processor_run(o_ins->processor,
+                                        0,
+                                        FLB_PROCESSOR_PROFILES,
+                                        evc->tag,
+                                        flb_sds_len(evc->tag),
+                                        (char *) profile_context,
+                                        0,
+                                        NULL,
+                                        NULL);
+
+                if (ret == 0) {
+                    ret = cprof_encode_msgpack_create(&serialized_profiles_context_buffer,
+                                                      profile_context);
+
+                    cprof_destroy(profile_context);
+
+                    if (ret != 0) {
+                        flb_coro_destroy(coro);
+                        flb_free(out_flush);
+                        flb_free(p_buf);
+
+                        return NULL;
+                    }
+
+                    if ((serialization_buffer_offset +
+                         cfl_sds_len(serialized_profiles_context_buffer)) > p_size) {
+                        resized_serialization_buffer = \
+                            flb_realloc(p_buf, p_size + cfl_sds_len(serialized_profiles_context_buffer));
+
```

Comment:
missing flb_errno();

Reply:
Addressed in [42f5d58](https://github.com/fluent/fluent-bit/pull/9748/commits/42f5d58e28867c391247fa73ea26302a1664f087)